src/style.css

```diff
@@ -1 +1,67 @@
 /* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+html {
+  max-width: 1200px;
+  min-width: 1024px;
+}
+
+.head {
```

Comment:
head not equal header

Reply:
what does you mean? 
i know that header and head is different. but i manage to class "head"